chainer/backends/_cpu.py

```diff
@@ -30,6 +30,9 @@ def __eq__(self, other):
     def __repr__(self):
         return '<{} (numpy)>'.format(self.__class__.__name__)
 
+    def __str__(self):
+        return '{}: @numpy'.format(self.__class__.__name__)
```

Comment:
Please do not include class name.
```suggestion
        return '@numpy'
```